pkg/ingester/client/client.go

```diff
@@ -64,7 +64,7 @@ func (c *closableHealthAndIngesterClient) Close() error {
 
 // Config is the configuration struct for the ingester client
 type Config struct {
-	GRPCClientConfig grpcclient.Config `yaml:"grpc_client_config" doc:"description=Configures the gRPC client used to communicate between distributors and ingesters."`
+	GRPCClientConfig grpcclient.Config `yaml:"grpc_client_config" doc:"description=Configures the gRPC client used to communicate with ingesters from distributors and queriers."`
```

Comment:
And rulers (when using the "internal" rule evaluation mode).

Reply:
> And rulers (when using the "internal" rule evaluation mode).

Also when writing data back to ingesters, no?

Reply:
Right.